<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Q_Interface</class>
 <widget class="QWidget" name="Q_Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>千词斩 Version 1.0</string>
  </property>
  <widget class="QPushButton" name="button_Exit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Register">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Supervisor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>我是权限狗</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_Login">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>千词斩 Version 1.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_About">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_More">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>更多</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
